alica_dummy_proxy/src/communication/AlicaDummyCommunication.cpp

```diff
@@ -216,12 +216,12 @@ AlicaDummyCommunication::AlicaDummyCommunication(const alica::AlicaCommunication
         : alica::IAlicaCommunication(callbacks)
         , _isRunning(false)
 {
-    s_modContainer.registerModule(this);
+    // s_modContainer.registerModule(this);
```

Comment:
What's going on here?

Reply:
@bjoernschroeder I have the same question actually

Reply:
It does seem related to https://github.com/rapyuta-robotics/alica/pull/356/files

Reply:
Yes, during the implementation, the bug we had with the logger appeared more often, so I added the fix we discussed with abhi. It should be equal to your PR dale